internal/rbd/controllerserver.go

```diff
@@ -1130,6 +1128,13 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 		return nil, err
 	}
 
+	// lock out volumeID for clone and delete operation
```

Comment:
why we are taking expandLock here for clone and delete ?

Reply:
This is not a new implementation, this is existing and the lock is moved below.